File: src/main/java/kr/hahaha98757/ihateplugin/IHatePlugin.java
package kr.hahaha98757.ihateplugin;

import org.bukkit.plugin.java.JavaPlugin;

public class IHatePlugin extends JavaPlugin {

    public static JavaPlugin instance;

    public IHatePlugin() {
        instance = this;
    }

    @Override
    public void onEnable() {
        PlayerTickEvent.register(this);
        getServer().getPluginManager().registerEvents(new EventListener(), this);
        getLogger().info("I Hate Plugin v2.0.0 enable");
    }

    @Override
    public void onDisable() {
        getLogger().info("I Hate Plugin v2.0.0 disable");
    }
}

File: src/main/kotlin/kr/hahaha98757/ihateplugin/IHatePlugin.kt
package kr.hahaha98757.ihateplugin

import org.bukkit.plugin.java.JavaPlugin

class IHatePlugin: JavaPlugin() {
    companion object {
        lateinit var instance: JavaPlugin
    }

    init {
        instance = this
    }

    override fun onEnable() {
        PlayerTickEvent.register(this)
        server.pluginManager.registerEvents(EventListener(), this)
        logger.info("I Hate Plugin v2.0.0 enable")
    }

    override fun onDisable() {
        logger.info("I Hate Plugin v2.0.0 disable")
    }
}
